src/worker/browser-env/runner/test-runner/execution-thread.ts

```diff
@@ -0,0 +1,49 @@
+import crypto from "node:crypto";
+import _ from "lodash";
+import NodejsEnvExecutionThread from "../../../runner/test-runner/execution-thread";
+import RuntimeConfig from "../../../../config/runtime-config";
+
+import { WorkerEventNames } from "../../types";
+
+import type { ViteWorkerCommunicator } from "../../communicator";
+import type { BrowserEnvRuntimeConfig } from "../../types";
+import type { ExecutionThreadCtorOpts } from "../../../runner/test-runner/types";
+import type { Test } from "../../../../test-reader/test-object/test";
+import type { Hook } from "../../../../test-reader/test-object/hook";
+
+export const wrapExecutionThread = (runUuid: string): typeof NodejsEnvExecutionThread => {
```

Comment:
Solution which gives me ability to pass `runUuid` and generate `ExecutionThread` class for browser. I use it because `ExecutionThread` class is used inside `NodejsEnvTestRunner` and I need some api to set it, but `ExecutionThread` from nodejs don't need this.